data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApiTypes.java

```diff
@@ -59,9 +67,15 @@ public class BeaconRestApiTypes {
   public static final ParameterMetadata<String> PARAMETER_BLOCK_ID =
       new ParameterMetadata<>(PARAM_BLOCK_ID, CoreTypes.string(PARAM_BLOCK_ID_DESCRIPTION, "head"));
 
-  public static final ParameterMetadata<UInt64> SLOT_PARAMETER =
+  public static final ParameterMetadata<UInt64> SLOT_QUERY_PARAMETER =
       new ParameterMetadata<>(SLOT, CoreTypes.UINT64_TYPE.withDescription(SLOT_QUERY_DESCRIPTION));
 
+  public static final ParameterMetadata<UInt64> SLOT_PATH_PARAMETER =
+      new ParameterMetadata<>(
+          SLOT,
+          CoreTypes.UINT64_TYPE.withDescription(
+              "The slot for which the block should be proposed."));
```

Comment:
this seems like a very specific description... we likely want something more generic in this definition, and override it if it needs something somewhere else... for this comment currently, it'll only work in 1-2 places..

Reply:
we could probably keep just SLOT_PARAMETER, and override description in the specific places the path_parameter is used

Reply:
also at some point we seem to have flipped naming from PARAMETER_XXX to XXX_PARAMETER if we're cleaning things up... i prefer PARAMETER first so they're grouped together in hints, but at least it would be good for it to be consistent..